### ISSUE TITLE
telegram-desktop color inversion of images and videos in fullscreen

### ISSUE THREAD

**Issue by dm17** [void-linux/void-packages#43786]
### Is this a new report?

Yes

### System Info

Void 6.2.13_1 x86_64-musl AuthenticAMD uptodate hold rFFF

### Package(s) Affected

telegram-desktop

### Expected behaviour

Image and video colors look the same in telegram-desktop as when they are fullscreened (anytime they're clicked on they full screen).

### Actual behaviour

The colors of images and videos invert in fullscreen mode

### Steps to reproduce

Click on any image or video, it will fullscreen, and you'll see different colors than in the original

**Comment by dm17** [comment]
@Johnnynator I opened this issue here first: https://github.com/telegramdesktop/tdesktop/issues/26264#issuecomment-1537371886
Then I realized it is probably distro-specific, as this tdesktop dev states, if it isn't present in their official static, then it is probably package/distro-specific.

**Comment by Johnnynator** [comment]
I can't reproduce this on any of my systems.

**Comment by dm17** [comment]
I'm using Xorg with AMDGPU.

**Comment by Johnnynator** [comment]
Neither of these are relevant for this issue and also get printed on a working configuration.

**Comment by dm17** [comment]
I just tried `4.8.1_1` - thanks to whomever just packaged it. No change. Yes, I'm a bit lost of potential causes. I don't have anything special with this system except for it being musl.

**Comment by dm17** [comment]
Fixed! The clue was two other programs I wasn't using that had the same inverted colors issue (regardless of fullscreen or not, in videos, no matter what driver or output method was used) - vlc and mplayer. After hitting my head on the table a few times, I figured I had 100% stock X11 configuration, but I found one setting was changed: `DefaultDepth`! So 10-bit color is generally broken on very standard hardware/software configurations in X11! Anyway, that's a bit of context if anyone gets stuck looking for the cause. `DefaultDepth 24` (8-bit) fixes this.
